hubblestack/utils/hashutils.py

```diff
@@ -24,7 +24,10 @@ def get_hash(path, form="sha256", chunk_size=65536):
         raise ValueError('Invalid hash type: {0}'.format(form))
 
     with hubblestack.utils.files.fopen(path, 'rb') as ifile:
-        hash_obj = hash_type()
+        if form == 'md5':
+            hash_obj = hash_type(usedforsecurity=False)    
+        else:
+            hash_obj = hash_type()
```

Comment:
Can we remove if else here? I mean is it ok if we always call **hash_type(usedforsecurity=False)**  irrespective of the value of **form**?

Reply:
done